From: Qingqing <wangyiqing@kylinos.cn>
Date: Mon, 17 Aug 2026 07:16:55 +0000
Subject: Translated using Weblate (Turkish)

Currently translated at 100.0% (5 of 5 strings)

Translation: openkylin-huanghe/peony-engrampa-menu-plugin
Translate-URL: http://weblate.openkylin.top/projects/openkylin-huanghe/peony-engrampa-menu-plugin/tr/
---
 peony-engrampa-menu-plugin/translations/peony-engrampa-menu_tr.ts | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/peony-engrampa-menu-plugin/translations/peony-engrampa-menu_tr.ts b/peony-engrampa-menu-plugin/translations/peony-engrampa-menu_tr.ts
index 76f3c89..b5a3d83 100644
--- a/peony-engrampa-menu-plugin/translations/peony-engrampa-menu_tr.ts
+++ b/peony-engrampa-menu-plugin/translations/peony-engrampa-menu_tr.ts
@@ -35,7 +35,7 @@
     <message>
         <location filename="../engrampa-menu-plugin.h" line="41"/>
         <source>Engrampa Menu Extension</source>
-        <translation type="unfinished"></translation>
+        <translation>Arşiv menüsü uzantısı</translation>
     </message>
     <message>
         <source>engrampa Menu Extension.</source>
@@ -44,17 +44,17 @@
     <message>
         <location filename="../engrampa-menu-plugin.cpp" line="74"/>
         <source>Compress</source>
-        <translation type="unfinished"></translation>
+        <translation>Sıkıştır</translation>
     </message>
     <message>
         <location filename="../engrampa-menu-plugin.cpp" line="90"/>
         <source>Uncompress to current path</source>
-        <translation type="unfinished"></translation>
+        <translation>Buraya çıkar</translation>
     </message>
     <message>
         <location filename="../engrampa-menu-plugin.cpp" line="101"/>
         <source>Uncompress to specific path...</source>
-        <translation type="unfinished"></translation>
+        <translation>Buraya çıkar...</translation>
     </message>
 </context>
 </TS>
